security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/BranchResult.java

```diff
@@ -12,9 +12,9 @@
 import java.util.Objects;
 
 /**
- * provide electrical information on a branch after a security analysis.
- * it belongs to pre and post Contingency results.
- * it is the result of the branch id specified in StateMonitor.
+ * Contains state variables of a branch. The state variables supported are active and reactive power at both sides,
```

Comment:
we could mention that it's always linked to a StateMonitor. `Contains state variables of a branch which is monitored during security analysis through a {@link StateMonitor}`